src/routes/notifications/v1/entities/__tests__/register-device.dto.builder.ts

```diff
@@ -10,19 +10,22 @@ export async function registerDeviceDtoBuilder(args?: {
   uuid?: UUID;
   cloudMessagingToken?: UUID;
   timestamp?: number;
+  timestampExpiryMinutes?: number;
 }): Promise<IBuilder<RegisterDeviceDto>> {
   const uuid = args?.uuid ?? (faker.string.uuid() as UUID);
   const cloudMessagingToken =
     args?.cloudMessagingToken ?? (faker.string.uuid() as UUID);
-  const timestamp = new Date(args?.timestamp ?? faker.date.recent());
-  timestamp.setMilliseconds(0);
-  const timestampWithoutMilliseconds = timestamp.getTime();
+  const currentDate = new Date(new Date().getTime());
+  const timestampExpiryMinutes = args?.timestampExpiryMinutes ?? 1;
+  currentDate.setMinutes(currentDate.getMinutes() + timestampExpiryMinutes);
```

Comment:
By adding the expiry, this means the signature would be inherently invalid. What was the reason to add it?

Reply:
Please have a look at the new version:
https://github.com/safe-global/safe-client-gateway/pull/2192/commits/6c61be2237ed10a4c56fd4509579f04bb923eac2